org.tenacityaudio.Tenacity.yaml

```diff
@@ -0,0 +1,270 @@
+app-id: org.tenacityaudio.Tenacity
+# freedesktop-sdk should work too, but causes problems with
+# the timeline mouse cursor on some systems for unkown reasons
+runtime: org.gnome.Platform
+runtime-version: '40'
+sdk: org.gnome.Sdk
+command: tenacity
+rename-desktop-file: tenacity.desktop
+rename-icon: tenacity
+rename-appdata-file: tenacity.metainfo.xml
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network # Tenacity uses network sockets for IPC
+  - --socket=pulseaudio
+  - --device=all # ALSA
+  - --filesystem=host
+  - --env=LD_LIBRARY_PATH=/app/lib/tenacity
+  - --env=ALSA_CONFIG_PATH=
+  - --env=LADSPA_PATH=/app/extensions/Plugins/ladspa
+  - --env=LV2_PATH=/app/extensions/Plugins/lv2
+  # Audacity uses VST_PATH
+  - --env=VST_PATH=/app/extensions/Plugins/lxvst
+  # JACK support via pipewire
+  - --filesystem=xdg-run/pipewire-0
+  - --system-talk-name=org.freedesktop.RealtimeKit1
+
+add-extensions:
+  org.audacityteam.Audacity.Codecs:
+    directory: lib/codecs
+    add-ld-path: lib
+    bundle: true
+    autodelete: true
+  org.freedesktop.LinuxAudio.Plugins:
+    directory: extensions/Plugins
+    version: '20.08'
+    add-ld-path: lib
+    merge-dirs: ladspa;lv2;lxvst
+    subdirectories: true
+    no-autodownload: true
+
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /share/aclocal
+  - /share/man
+  - '*.la'
+  - '*.a'
+modules:
+
+  - name: wxwidgets
+    rm-configure: true
+    config-opts:
+      - --with-libpng
+      - --with-zlib
+      - --with-cxx=14
+      - --disable-sdltest
+      - --disable-webview
+      - --disable-webviewwebkit
+      - --disable-ribbon
+      - --disable-propgrid
+      - --disable-richtext
+      - --with-expat=builtin
+      - --with-libiconv=/usr
+    cleanup:
+      - /share/bakefile
+    sources:
+      - type: git
+        url: https://github.com/wxWidgets/wxWidgets
+        tag: v3.1.5
+        commit: 9c0a8be1dc32063d91ed1901fd5fcd54f4f955a1
+
+  - deps/lame-3.100.json
+  - deps/libid3tag/libid3tag.json
```

Comment:
Replace this with the new fork:
https://github.com/flathub/org.mixxx.Mixxx/blob/master/org.mixxx.Mixxx.yaml#L118

Reply:
What about the patches located at https://github.com/TheEvilSkeleton/flatpaks/tree/org.tenacityaudio.Tenacity/deps/libid3tag, do I keep them or remove them?

Reply:
I merged them into the fork. You can get rid of them.